Detector/DetComponents/src/MaterialScan.h

```diff
@@ -28,4 +28,6 @@ class MaterialScan: public Service {
   double m_etaBinning;
   /// Maximum eta until which to scan, scan is performed from -m_etaMax to +m_etaMax
   double m_etaMax;
+  /// Number of bins for scan, scan is performed from with binning m_phi between -m_etaMax +m_etaMax
+  double  m_phi;
```

Comment:
Why is this member only added in the header? Would expect corresponding changes in the source, was this added by accident or did you forget to add the source-changes? Also the comment still says "eta".